src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -95,4 +95,8 @@
                 CAN.Swerve.REAR_RIGHT_DRIVE_MOTOR_ID, DRIVE_MOTOR_CONFIG.withPID(
                 LOCAL_SWERVE_MODULES.rearRightModuleConstants.driveCoefs));
     }
+    protected static class TransporterComponents{
+        public static final TrigonTalonSRX MOTOR = SwerveComponents.FRONT_RIGHT_ENCODER;
```

Comment:
shouldn't swerve take from transporter due to motorconfig? if not have a motor config effect the motor

Reply:
This is the way we did it for the rest... it should be fine... -Natan